website/docs/prysm-usage/checkpoint-sync.md

```diff
@@ -4,72 +4,218 @@ title: Sync from a checkpoint
 sidebar_label: Sync from a checkpoint
 ---
 
+import CheckpointSyncPresent from '@site/static/img/checkpoint-sync-present.png';
+import CheckpointSyncAbsent from '@site/static/img/checkpoint-sync-absent.png';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';
 
-<HeaderBadgesWidget />
+<HeaderBadgesWidget commaDelimitedContributors="Kasey,Mick,James" lastVerifiedDateString="August 18th, 2022" lastVerifiedVersionString="v2.1.4" />
 
 :::caution
 
 **This feature is currently in public preview** and may change significantly as we receive feedback from users like you. Join our [Discord server](https://discord.gg/prysmaticlabs) to share your feedback.
 
 :::
 
-Prysm provides the ability to sync from a finalized checkpoint, as an alternative to replaying all history starting from Genesis. Checkpoint Sync is significantly faster than Genesis Sync, and is considered more secure thanks to the protections against long-range attacks afforded by [Weak Subjectivity](https://blog.ethereum.org/2014/11/25/proof-stake-learned-love-weak-subjectivity/).
+**Checkpoint sync** is a feature that significantly speeds up the initial sync between your beacon node and the Beacon Chain. With checkpoint sync configured, your beacon node will begin syncing from a recently finalized checkpoint instead of syncing from genesis. This can make installations, validator migrations, recoveries, and testnet deployments *way* faster.
+
+This how-to walks you through two checkpoint sync configuration methods: syncing via **network**, and syncing via **file**. After configuring checkpoint sync, we strongly recommend [verifying the authenticity of your beacon node's checkpoint](#verify-the-authenticity-of-your-beacon-nodes-checkpoint) as a way to "trust but verify" the integrity of your checkpoint data.
```

Comment:
Can we add a comment recommending networking syncing? Mainly so that users don't run into trouble trying to install prysmctl. Like "For most use cases, networking syncing will be the simpler method and is recommended."